src/main/java/mate/academy/onlinebookstore/OnlineBookStoreApplication.java

```diff
@@ -1,13 +1,40 @@
 package mate.academy.onlinebookstore;
 
+import java.math.BigDecimal;
+import mate.academy.onlinebookstore.model.Book;
+import mate.academy.onlinebookstore.service.BookService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.CommandLineRunner;
 import org.springframework.boot.SpringApplication;
 import org.springframework.boot.autoconfigure.SpringBootApplication;
+import org.springframework.context.annotation.Bean;
 
 @SpringBootApplication
 public class OnlineBookStoreApplication {
+    @Autowired
+    private BookService bookService;
 
     public static void main(String[] args) {
         SpringApplication.run(OnlineBookStoreApplication.class, args);
     }
 
+    @Bean
+    public CommandLineRunner commandLineRunner() {
+        return new CommandLineRunner() {
+            @Override
+            public void run(String... args) throws Exception {
+                Book book = new Book();
+                book.setCoverImage("image");
+                book.setIsbn("Ranok");
+                book.setDescription("About nature");
+                book.setTitle("From nature");
+                book.setPrice(BigDecimal.valueOf(500));
+                book.setAuthor("Stiven");
+
+                bookService.save(book);
+
+                System.out.println(bookService.findAll());
+            }
+        };
```

Comment:
use lambda (not anonymous class)
```suggestion
        return new CommandLineRunner() {
            @Override
            public void run(String... args) throws Exception {
                Book book = new Book();
                book.setCoverImage("image");
                book.setIsbn("Ranok");
                book.setDescription("About nature");
                book.setTitle("From nature");
                book.setPrice(BigDecimal.valueOf(500));
                book.setAuthor("Stiven");

                bookService.save(book);

                System.out.println(bookService.findAll());
            }
        };
```